corehq/apps/hqwebapp/templates/hqwebapp/partials/form_list_form.html

```diff
@@ -25,7 +25,7 @@
         </thead>
 
         <tbody data-bind="foreach: rows">
-            <tr>
+            <tr data-bind="css: { error: Object.keys(errors).length !== 0 }">
```

Comment:
yea it's pretty bad. don't know why i didn't mention this before, but underscore has a method for just (that)[http://underscorejs.org/#isEmpty] `_.isEmpty`